src/components/accordion/accordion.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <div class="ui-accordion" role="heading" :class="rootClasses">
```

Comment:
Is the `role="heading"` really on this element? The body slot is also within it.

Reply:
i have removed the accessibility attributes for now